src/main/java/com/example/nlern/models/Test.java

```diff
@@ -1,61 +1,30 @@
 package com.example.nlern.models;
-import jakarta.persistence.*;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import jakarta.persistence.Id;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.ManyToOne;
+
+import lombok.Getter;
+import lombok.Setter;
 
 @Entity
 @Table(name = "tests")
+@Getter @Setter
 public class Test {
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private int id;
-    private String name;
 
+    private String name;
     short duration;
```

Comment:
1) no access modifier is really bad
2) why exactly do we need to use short here?

Reply:
we use short because passing the exam cannot be longer than 30000 minutes

Reply:
so 29999 minutes is okay?

Reply:
If you intent to have time constraints for duration, do so by providing proper validation, not by downsizing primitive type